extensions/big-data-cluster/src/wizards/create-cluster/pages/targetClusterPage.ts

```diff
@@ -126,7 +127,9 @@ export class SelectTargetClusterPage extends WizardPageBase<CreateClusterModel>
 
 			configFileInput.value = fileUri.fsPath;
 
-			let clusters = self.model.loadClusters(fileUri.fsPath);
+			await setActiveKubeconfig(fileUri.fsPath);
```

Comment:
Minor: shouldn't it be `setActiveKubeConfig`? Or is `Kubeconfig` one word with no camel casing?

Reply:
We're using one word with no camel case through out the kubectl calls.